EdgeOfEmpireBot/EdgeOfEmpireBot/MessageHandlers/SteamHandler/SteamMessageHandler.cs

```diff
@@ -5,35 +5,43 @@ namespace HK47.MessageHandlers;
 
 public class SteamMessageHandler(ISteamService steamService, IDataService dataService, IMessageService messageService) : ISteamMessageHandler
 {
-    /// </inheritdoc>
+    /// <inheritdoc/>
     public async Task ProcessCommand(string userInput)
     {
         var commandParams = userInput.Split(' ');
-        var command = commandParams!.First().ToLower();
+        var command = commandParams[0].ToLower();
         switch (command)
         {
-            case "requestgame":
-            case "gr":
-                try
-                {
-                    var appId = commandParams[1];
-                    var game = await steamService.RetrieveGameFromSteam(appId);
-                    await dataService.WriteGameToFile(game);
-                    await messageService.SendMessage($"```[Statement]: {game.Name} was added to the list.```");
-                }
-                catch (Exception ex)
-                {
-                    await messageService.SendMessage($"[Error]: Game not added due to the following:\n{ex.Message}");
-                }
+            case "requestGame":
```

Comment:
not sure if its just github UI I'm not use to, but is there no longer case gr: and just requestGame?

Reply:
Oh yeah I was going to bring this up but completely forgot to.
I was thinking that having multiple aliases for commands can make things extremely messy. Since the purpose of the issue was to clean up the switch statements I thought this was an area in which some cleanup was necessary

Reply:
I don't know, I like aliases, switch with a handful of cases for something isn't bad, I think the cleanup was around having  additional within the switch.


Maybe it's something we need to improve with the router itself? Ie mapping aliases to the main command?

Reply:
using aliases in the switch isnt bad if the amount of use cases is low. However, it will inevitably lead to messy code if aliases are there for more than 5 commands. In terms of improving the router with mapping aliases to the main command then I can see only a couple ways to keep this clean. We can either

1. Have a list for each command that includes its aliases then do a .Contains() on the list and pass through the command that way.
2. Adjust the key value pair in the commands.json (What I have implemented)

I am interested in the approaches you have thought about though.